### PR TITLE
Update __init__.py

### PR DESCRIPTION
**Pull request by b-aeby**
Replace deprecated hass.config_entries.async_setup_platforms(entry, PLATFORMS)

### CODE REVIEW

File: custom_components/awox/__init__.py [hunk 1]
```diff
@@ -35,7 +35,8 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     # Make `mesh` accessible for all platforms
     hass.data[DOMAIN][entry.entry_id] = mesh
 
-    hass.config_entries.async_setup_platforms(entry, PLATFORMS)
+    # DEPRECATED: hass.config_entries.async_setup_platforms(entry, PLATFORMS)
```
Comment (fsaris) [inline custom_components/awox/__init__.py:38]: Tnx for the PR

This line can be removed. No need to leave the depredation comment. That's what version control is for

**Comment by fsaris** [comment]
@b-aeby you closed the PR. Isn't this the correct solution?

**Comment by b-aeby** [comment]
Hi,
PR re-open.
It solves this issue but the devices are not consistently available. I'm using RPi4, it has already worked properly but not usable after a SD crash and full re-installation.

Regards
Benjamin

15 mai 2023 09:26 "Frans Saris" ***@***.*** ***@***.******@***.***>)> a écrit:
	@b-aeby (https://github.com/b-aeby) you closed the PR. Isn't this the correct solution?

**Comment by b-aeby** [comment]
It works fine with a usb bluetooth dongle!
15 mai 2023 09:26 "Frans Saris" ***@***.*** ***@***.******@***.***>)> a écrit:
	@b-aeby (https://github.com/b-aeby) you closed the PR. Isn't this the correct solution?

**Pull request closed without merging**